src/styles/blocks/card.scss

```diff
@@ -0,0 +1,97 @@
+.card {
+  padding: 32px 16px 16px;
+  width: $card-width;
+
+  border: 1px solid $border-color;
+  border-radius: 5px;
+
+  &__image {
+    display: block;
+    width: 160px;
+    height: 134px;
```

Comment:
There will be good idea to add [objec-fit](https://developer.mozilla.org/en-US/docs/Web/CSS/object-fit) property for image. This property control how image will be scaled in size of your image will be different from size of your container

Reply:
Added object-fit: cover ✅